nixos/modules/services/networking/cjdns.nix

```diff
@@ -263,4 +263,8 @@ in
 
   };
 
+  meta.tests = {
+    "cjdns" = ./tests/cjdns.nix;
```

Comment:
This is also inconsistent with the rest of the attributes where the test-name is without quotation.